nativesintech/MMIWG2S
# iOS 29 Add back/share buttons, haptics, and visibility changes to ARFaceViewUI.

Added back and share buttons.
Add haptic feedback to all buttons in the ARFaceViewUI.
Add visibility changes for when buttons are tapped and when activity controller closes.

To test:
Start the app, go to the ARView.
Tap the color buttons and feel the haptics.
Tap the Capture button. The color should change to black and you should feel haptics. All the views should disappear.
Back and Share should appear.
Tapping Back should bring back the other hidden buttons and hide Back and Share.
Tapping Share should bring up the activity controller. When you close the activity controller, the capture and color buttons should come back, while hiding the Back and Share button.
